### DISCUSSION TITLE
Swap Archived tile for Offline tile on devices page

### DISCUSSION THREAD

**Discussion by iamtheonlydave** [netalertx/NetAlertX#705] [Ideas]
A quality of life improvement for me would be if the 6th tile on devices page could be Offline instead of Archived.

I mainly access NetAlertX on my mobile so I understand the reason for only having 6 tiles, My Devices, Connected, Favorites, New Devices, Down, Archived. But in order to see Offline devices I have to do 2 presses on small letters (Devices then Offline) in the left panel which is not so easy as tapping one tile. 

For me Offline is a higher priority piece of information than Archived (or Favorite) but I appreciate there may be other opinions.

**Comment by jokob-sk** [comment]
![image](https://github.com/jokob-sk/NetAlertX/assets/96159884/f0748a12-56b8-4c01-957c-e6bc59bcdf58)

This should be available in the next release. Please have a look at the `netalertx-dev` docker image, in about 15 minutes (or after the last [action finishes](https://github.com/jokob-sk/NetAlertX/actions)) from now. 

👉If I don't get a confirmation that this is working I might pull out the feature to prevent bugs. 

Make sure you refresh your browser cache - and click the 🔄 refresh button in the top right corner.

It would be great if you could test this (backup everything first or use a new container) on your end by switching to the above image and letting me know if the issue was resolved/the new feature is working.

⭐ This feature took 3h ⏳ to implement - consider becoming a sponsor ⭐

Thanks in advance,
j

**Reply by iamtheonlydave** [reply]
Thank you @jokob-sk I can confirm that it works for the one use case I have tested - 4 selections my_devices, connected, down, offline

And thank you for all you work. I did a single donation some time back but am likely to do more as time goes by and my funds allow (not an excuse but I'm a pensioner. Actually that is an excuse)
Luv d

**Reply by jokob-sk** [reply]
Hey, don't feel like you have to donate, if you can share the app with others, then even better. Thank you for your support! 🙏Glad to hear it's working 🙂
